### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by mirouhml**
- Match the design with [Zeplin](https://zeplin.io/)'s design.
- Add random functionality for progress, chapters and current chapter since the bookstore API doesn't cover that.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,11 +1,12 @@
-@import url('https://fonts.googleapis.com/css2?family=Roboto');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;700&display=swap');
```
Comment (Meltrust) [inline src/App.css]: Your project imports and uses the Montserrat font family, but not the RobotoSlab which is required in several elements:

image

Therefore:

- [ ] Please, import and implement the RobotoSlab font family where it's required, according to the Zeplin.  Please remember to check that the exact font styling matches each element.

That way, your project will look like the Zeplin, and we will meet this requirement. 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @mirouhml,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Nice general styling! ✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 1 above)
Comment (mirouhml) [reply src/App.css]: Thanks for catching that!

Comment (Whoistolu) [reply src/App.css]: All good now 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @mirouhml 

Thank you for implementing the recommended changes.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mirouhml** [comment]
Thanks for the approval!

**Pull request merged**